app/src/main/java/com/infomaniak/mail/utils/UiUtils.kt

```diff
@@ -37,6 +37,7 @@ import com.infomaniak.mail.utils.extensions.updateNavigationBarColor
 object UiUtils {
 
     const val FULLY_SLID = 1.0f
+    const val IGNORE_DIVIDER_TAG = "ignoreDividerTag"
```

Comment:
Should this not have gone into the DividerItemDecorator class instead?

Reply:
The `DividerItemDecorator` class is in Core, and doesn't use `IGNORE_DIVIDER_TAG`.
So I think not.

Reply:
Ah yeah it takes a lambda to determine on what condition to ignore, ok, yeah, UiUtils is probably the best place indeed